buildSrc/src/main/java/org/elasticsearch/gradle/DependenciesGraphPlugin.java

```diff
@@ -42,22 +42,37 @@ public void apply(Project project) {
                 t.setUrl(url);
             });
         });
+        project.getRootProject().getPluginManager().apply(DependenciesGraphHookPlugin.class);
```

Comment:
nit: can we do this as the very first line in the `apply()` method? This is a somewhat unwritten convention.